recipes/stx/all/conandata.yml

```diff
@@ -0,0 +1,14 @@
+sources:
+    "1.0.0":
```

Comment:
I would suggest removing this version since, the higher one is near identical

Reply:
sounds good to me